### PR TITLE
PYTHON-1377 OP_MSG unacknowledged write support

### PR DESCRIPTION
**Pull request by ShaneHarvey**
Fixes MockupDB unacknowledged OP_MSG support.

### CODE REVIEW

File: mockupdb/__init__.py [hunk 1]
```diff
@@ -672,6 +672,8 @@ def command_name(self):
             return list(self.docs[0])[0]
 
     def _replies(self, *args, **kwargs):
+        if self.flags & OP_MSG_FLAGS['moreToCome']:
+            assert False, "Cannot reply to OpMsg with moreToCome: %r" % (self,)
```
Comment (ajdavis) [inline mockupdb/__init__.py]: cool.

File: mockupdb/__init__.py [hunk 2]
```diff
@@ -1207,19 +1209,12 @@ def __init__(self, port=None, verbose=False,
         self._autoresponders = []
 
         if auto_ismaster is True:
-            self.autoresponds(Command('ismaster'),
+            self.autoresponds(CommandBase('ismaster'),
```
Comment (ajdavis) [inline mockupdb/__init__.py]: Curious: Does PyMongo ever send isMaster over OP_MSG?

Comment (ShaneHarvey) [reply mockupdb/__init__.py]: Yes, the initial handshake uses OP_QUERY then if maxWireVersion>=6 all subsequent commands on the socket use OP_MSG.

**Comment by ShaneHarvey** [comment]
Can you merge this as soon as you can? I don't have write permission on this repo. Thanks!

**Comment by ajdavis** [comment]
Done, LMK if you want a MockupDB release or if you're using it from master.

**Pull request merged**